build.sbt

```diff
@@ -27,8 +27,8 @@ val settings = Common.settings ++ Common.publish ++ Seq(
   credentials ++= Common.credentials(),
   libraryDependencies ++= Seq(
     "org.scalatest"     %%% "scalatest"       % "3.0.5" % "test",
-    "com.github.amlorg" %%% "amf-webapi"      % "3.2.1",
-    "com.github.amlorg" %%% "amf-validation"  % "3.2.1"
+    "com.github.amlorg" %%% "amf-webapi"      % "3.2.2",
```

Comment:
latest is [3.3.0](https://github.com/aml-org/amf/releases/tag/3.3.0)

Reply:
@jstoiko can we make this upgrade in a separate PR?
Release 3.3.0 adds breaking changes (section "Interface Changes") which may break things in webapi-parser code and/or examples. There will be too much in this PR if we fix migration issues here.

Reply:
Also see https://github.com/aml-org/amf/issues/542